plugins/scheduler/README.md

```diff
@@ -0,0 +1,87 @@
+# Vanilla Scheduler
+
+Vanilla Scheduler is a vanilla/vanilla add-on that adds a Message Queue functionality.
+
+In vanilla/vanilla context, a Message Queue allows postponing the execution of a job after the response is processed and sent to the user or even to delegates it to a third-party player for processing.
+
+The main goal of Vanilla Scheduler is to define the different interfaces of operation to accomplish such a task and provided a basic implementation of those interfaces to have an out-of-the-box functionality available.
+
+Vanilla Scheduler add-on is built as a three-layer component:
+- Scheduler: The Scheduler is event-based and takes jobs during normal request processing, and dispatch them at the end of the request.
+- Driver: The Driver is the component that knows how to handle a Job.
+- Job: The job is the actual work that needs to be done. Depending on the kind of job, this could mean the code to run (let's think in a Local Driver implementation) or the information needed to run the job on a remote third-party queue system.
+
+## Default implementations
+
+### DummyScheduler - The default Scheduler implementation
+DummyScheduler is actually a `first-in-first-out after-response event-fired scheduler`.
+Which means  the following:
+- Jobs are scheduled during normal request processing.
+- At the end of the request, an event is fired instructing the Scheduler to dispatch the Jobs.
+- Jobs are dispatched one by one in the order they were added to the Scheduler.
```

Comment:
We should state that there's no guarantee is will stay like that.
That way people won't code 2 jobs being co-dependent by relying on this.